### ISSUE TITLE
Pod's only show cpu/mem request+limit, missing usage

### ISSUE THREAD

**Issue by jasperslot** [lensapp/lens#291]
**Describe the bug**
Install Prometheus from within Lens. Go to the settings of the cluster and use the install button right below Metrics. This cluster has no Prometheus installed before. Once this is installed, go to the Pod's section and click on one of the pods. Now you see a CPU graph with 'requests' & 'limits', where it also should show 'usage'.

**To Reproduce**
1. Go to Pods
2. Click on a Pod
3. Click on CPU or Memory
4. You will only see a graph with the values 'Requests' & 'Limits'.

**Expected behavior**
As I have seen in the video and in some screenshots, it should also show 'Usage', where I am actually interested in, but for some reason it doesn't.

**Environment (please complete the following information):**
- Lens Version: 3.3.1
- OS: Windows 10

**Comment by james-s-nduka** [comment]
Any updates on this? I am using my own install of prometheus operator and CPU usage is not being displayed

**Comment by jasperslot** [comment]
Two days ago I upgraded one of our AKS clusters from Kubernetes 1.15.11 to 1.17.9 and magically the usage started working within Lens :) So maybe it has something to do with specific versions of Kubernetes or in my case AKS?

**Comment by james-s-nduka** [comment]
I am using EKS and the cluster is at version 1.17. I imagine it might be due to the configuration of the Prometheus operator I’m using?

**Comment by nevalla** [comment]
@james-s-nduka how did you install your prometheus-operator?

**Comment by jasperslot** [comment]
Could be, I have installed Prometheus from within Lens...

**Comment by james-s-nduka** [comment]
@nevalla I have installed via the Helm chart. Also in a Namespace I’ve called “monitoring”

**Comment by nevalla** [comment]
From what chart? I'll try to reproduce this.

**Comment by james-s-nduka** [comment]
The latest “stable/Prometheus” chart

**Comment by nevalla** [comment]
> The latest “stable/Prometheus” chart

AFAIK `stable/prometheus` does not install prometheus-operator. Have you tried to change prometheus installation method in cluster settings to Auto detect or Helm?

**Comment by james-s-nduka** [comment]
I’ve changed it to “Prometheus Operator” and pointed it at “monitoring/prometheus-server:80” in the Lens settings

**Comment by james-s-nduka** [comment]
Sorry is there a difference between “prom/prometheus” image and prometheus operator? I have a deployment for prometheus as part of that Helm chart.

Again everything is being shown in Lens apart from CPU usage

**Comment by nevalla** [comment]
Yes they are storing metrics data slightly differently. Are cpu metrics shown with `Auto detect` option?

**Comment by james-s-nduka** [comment]
With "auto detect" I get no metrics in Lens now and see;

```
Metrics are not available due to missing or invalid Prometheus configuration.

Right click cluster icon to open cluster settings.
```

**Comment by james-s-nduka** [comment]
I've just got it to work. 
I needed to select `Helm` as the Prometheus installation method 👍

**Comment by brandoncollins7** [comment]
I have been having a similar issue for a while, but it doesn't seem to be related to my prometheus, as everything works if I use "Helm" or autodetect, except for one spot. When I click on a node, and it lists each pod on that node, none of the cpu/mem sections are missing their data. However when I click on the running pod I do see it's full metrics on it's graph. Every other metric seems to be displaying fine. Any idea what could cause them to not show up only in this one area?

![image](https://user-images.githubusercontent.com/36413482/120032843-ca1a0780-bfc8-11eb-8ee2-a6017a9b4446.png)


All fine here:

![image](https://user-images.githubusercontent.com/36413482/120032713-96d77880-bfc8-11eb-9bc4-38bcde584eb7.png)

**Comment by Nokel81** [comment]
@brandoncollins7 that one section on the nodes detail page is from kube metrics (not prometheus) though I don't know why it wouldn't be give back results.

**Comment by jxlarrea** [comment]
@Nokel81 I'm having the same issue as @brandoncollins7, the list of pods at the node level are not showing CPU and Memory usage. If I click on a pod all usage metrics are properly displayed. 

Pretty sure this was working properly before updating to the latest version.

**Comment by Nokel81** [comment]
> @Nokel81 I'm having the same issue as @brandoncollins7, the list of pods at the node level are not showing CPU and Memory usage. If I click on a pod all usage metrics are properly displayed. 
> 
> Pretty sure this was working properly before updating to the latest version. 

Looking at the code history, I don't think it was ever working. But I might be wrong. We will want to get it to work though

**Comment by brandoncollins7** [comment]
> > @Nokel81 I'm having the same issue as @brandoncollins7, the list of pods at the node level are not showing CPU and Memory usage. If I click on a pod all usage metrics are properly displayed.
> > Pretty sure this was working properly before updating to the latest version.
> 
> Looking at the code history, I don't think it was ever working. But I might be wrong. We will want to get it to work though

It definitely used to work. I found this to be a greatly valuable feature in easily determine which pod is causing my cpu usage to be higher then normal. Now I am being forced to click on each pod one by one to view their cpu graph individually until I find the culprit.

**Comment by Nokel81** [comment]
Okay, we will look into adding this to our 5.0.1 or 5.1 release.

**Comment by brandoncollins7** [comment]
> Okay, we will look into adding this to our 5.0.1 or 5.1 release.

I wanted to try this. But given it's beta state, I cancelled the installation because I'm worried it'll mess up my existing setup. How safe is it to install this, does it wipe out my 4 installation?

**Comment by Nokel81** [comment]
It should be pretty safe to upgrade, but to be on the safe side I would copy your [`userData`](https://www.electronjs.org/docs/api/app#appgetpathname) folder to a different location.

**Comment by brandoncollins7** [comment]
> It should be pretty safe to upgrade, but to be on the safe side I would copy your [`userData`](https://www.electronjs.org/docs/api/app#appgetpathname) folder to a different location.

Is that {user}/AppData/Roaming/Lens on Windows?

**Comment by Nokel81** [comment]
`%APPDATA%/Lens` so yes I think

**Comment by brandoncollins7** [comment]
I tried it. Looks like this issue is resolved in Lens 5!! :D 

I've also added a feature request here, would be very beneficial to see a column added for requested cpu/memory along with the actual usage, do you think this would be possible? https://github.com/lensapp/lens/issues/3024

**Comment by brandoncollins7** [comment]
Looks like the log viewer is no longer accessible:( Is this expected? 

![image](https://user-images.githubusercontent.com/36413482/121572724-7b219880-c9f2-11eb-8dff-fd858cf866e4.png)

**Comment by brandoncollins7** [comment]
I've noticed the log viewer is no longer accessible from the pods screen:( Is this expected?

![image](https://user-images.githubusercontent.com/36413482/121572974-b91ebc80-c9f2-11eb-98a2-01c4f280ed4b.png)

![image](https://user-images.githubusercontent.com/36413482/121573226-01d67580-c9f3-11eb-9cc1-217d86e75fe5.png)

**Comment by Nokel81** [comment]
That is a known bug with the most recent beta release. We should have a new beta release out soon that resolves that.

**Comment by Nokel81** [comment]
Could this be fixed by #3653 ?
